backend/dataall/modules/dataset_sharing/services/share_processors/s3_access_point_process_share.py

```diff
@@ -215,16 +215,20 @@ def clean_up_share(
             source_env_group,
             env_group,
         )
-        clean_up = clean_up_folder.delete_access_point(share=share, dataset=dataset)
-
-        if clean_up:
-            clean_up_folder.delete_target_role_access_policy(
-                share=share, dataset=dataset, target_environment=target_environment
-            )
-            if not dataset.imported or dataset.importedKmsKey:
-                clean_up_folder.delete_dataset_bucket_key_policy(dataset=dataset)
-
-        return True
+        log.info("##### Cleaning up folder share resources #######")
```

Comment:
This is a quick fix - we may want to make folder revoke better and add the clean up steps as part of the `def process_revoked_share()` 

Otherwise, revoking a folder item may not clean up all of the resources before reaching a `REVOKE_SUCCEEDED` state

For now leaving design as is and just implementing fix so we don't run into larger issues when upgrading DA and revoking items